builder/dockerfile/imagecontext.go

```diff
@@ -12,30 +12,31 @@ import (
 	"github.com/pkg/errors"
 )
 
+type contextsBackend interface {
+	imageMounter() builder.ImageMounter
+	pullOrGetImage(name string) (builder.Image, error)
+}
+
 // imageContexts is a helper for stacking up built image rootfs and reusing
 // them as contexts
 type imageContexts struct {
-	b      *Builder
 	list   []*imageMount
 	byName map[string]*imageMount
 	cache  *pathCache
 }
 
-func (ic *imageContexts) new(name string, increment bool) (*imageMount, error) {
```

Comment:
The call that was using `increment` was replaced with a call to `newImageMountWithID()`